http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/AppComponent.scala

```diff
@@ -140,8 +141,8 @@ class AppTable(tag: Tag) extends Table[AppRecord](tag, "APP") {
           chart,
           release,
           samResourceId,
-          googleServiceAccount,
-          kubernetesServiceAccount,
+          serviceAccounts._1,
+          serviceAccounts._2,
```

Comment:
This PR hit scala 22 parameter limit, so I have to group those two into one, same for Autodelete